clients/cli/src/edit.rs

```diff
@@ -3,28 +3,27 @@ use std::io::Write;
 
 use lockbook_core::Core;
 use lockbook_core::Error as LbError;
-use lockbook_core::GetFileByPathError;
+use lockbook_core::FileType::Document;
```

Comment:
Same as above, I don't we should import variants.

Reply:
db4d3181